sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -853,9 +852,11 @@ class MultiFileParquetPartitionReader(
     false
   }
 
-  override def calculateEstimatedBlocksOutputSize(currentChunkedBlocks: Seq[DataBlockBase],
+  override def calculateEstimatedBlocksOutputSize(
+      filesAndBlocks: LinkedHashMap[Path, ArrayBuffer[DataBlockBase]],
       schema: SchemaBase): Long = {
-    calculateParquetOutputSize(currentChunkedBlocks, schema, true)
+    val allBlocks = filesAndBlocks.flatMap(item => item._2).toSeq
```

Comment:
nit: I think this can just be filesAndBlocks.values.flatten.toSeq

Reply:
thx, Done